front/components/assistant_builder/ActionsScreen.tsx

```diff
@@ -704,6 +704,7 @@ function ActionEditor({
                     <Input
                       name="actionName"
                       placeholder="My tool name…"
+                      disabled={action.noConfigurationRequired}
```

Comment:
I would use it for `shouldDisplayAdvancedSettings` instead?

Reply:
Maybe `canEditName` to be more specific ?